search_engine.py

```diff
@@ -0,0 +1,51 @@
+import sqlite3
```

Comment:
You should not have any direct interaction with the DB, it should only use the ORM.

Reply:
the search engine file is not relevant for now, idk why it's in this PR.